### ISSUE TITLE
not working in react

### ISSUE THREAD

**Issue by dhana6795** [EddieLa/JOB#62]
i found this is available in npm and tried to use in my react app it is installed but not working, i tried  this in html works well, i want to use this in react app, can @EddieLa  is there any possibility that you can fix the source which is available in npm

**Comment by ankurbabu** [comment]
Hi @dhana6795 ,

Did you able to use the library with Reactjs?

**Comment by dhana6795** [comment]
Yes, but not directly, i've added barcode reader object to window object then accessed

**Comment by ankurbabu** [comment]
Can you give the npm link of this project?

**Comment by dhana6795** [comment]
@ankurbabu  https://www.npmjs.com/package/barcode-reader

**Comment by ankurbabu** [comment]
@dhana6795 Thank you so much. Could you please tell me which method did you use to receive the image as input for decoding?

**Comment by ankurbabu** [comment]
@dhana6795 Thank you so much. But when I am using it in my code, I am receiving following error with Init() method "barcode_reader__WEBPACK_IMPORTED_MODULE_19___default.a.Init is not a function". Could you please tell how did you add barcode reader object to window object and used.
